tests/test_algo_common.py

```diff
@@ -143,7 +143,7 @@ def __str__(self):
     CompressionLevelTestStruct(
         config_provider=TestConfigCreator().add_algo('filter_pruning', {
             'num_init_steps': 1,
-            'pruning_steps': 1,
+            'pruning_steps': 2,
```

Comment:
This test was updated because I changed the pruning schedule. From documentation
> `pruning_steps` - the number of epochs during which the pruning rate target is increased from `pruning_init` to `pruning_target` value.

i.e. on epoch with index `pruning_steps - 1` we should set `pruning_target` value.
It was not true before my PR. I fixed it.